src/iframeWrapper.ts

```diff
@@ -149,7 +151,7 @@ export default class IframeWrapper {
   }
 
   private createWidgetBubble() {
-    if (this.walletType === WalletTypes.Full) {
+    if (this.walletType === AppMode.Full) {
```

Comment:
Should this check appMode?

```suggestion
    if (this.appMode === AppMode.Full) {
```

Reply:
Yep, changed `this.walletType` Type to `WalletType` and fixed some other errors, replaced other usage of this.walletType to this.appMode.